qt/python/mantidqtinterfaces/mantidqtinterfaces/TexturePlanner/helpers/test/test_detector_geometry.py

```
@@ -0,0 +1,256 @@
+    def test_starting_ind_zero_when_no_null_group(self, mock_load_grp):
+        dg, _ = self._make_dg_with_stubs()
+        tmp_grp = MagicMock()
+        tmp_grp.extractY.return_value = np.array([[1], [2], [3]])
+        mock_load_grp.return_value = tmp_grp
+
+        dg.recompute()
+
+        self.assertEqual(dg.starting_ind, 0)
+
+    def test_captures_det_positions_from_starting_ind(self, mock_load_grp):
+        dg, group_ws = self._make_dg_with_stubs(n_hist=4)
```

Ruff (0.15.17)

[warning] Unpacked variable `group_ws` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)